internal/params/config.go

```diff
@@ -9,143 +9,33 @@ import (
 
 // Genesis hashes to enforce below configs on.
 var (
-	MainnetGenesisHash = common.HexToHash("0xd4e56740f876aef8c010b86a40d5f56745a118d0906a34e69aec8c0db1cb8fa3")
-	TestnetGenesisHash = common.HexToHash("0x41941023680923e0fe4d74a34bdac8141f2540e3ae90623718e47d66d1ca4a2d")
-	RinkebyGenesisHash = common.HexToHash("0x6341fd3daf94b748c72ced5a5b26028f2474f5f00d824504e4fa37a75767e177")
-	GoerliGenesisHash  = common.HexToHash("0xbf7e331f7f7c1dd2e05159666b3bf8bc7a8a3a9eb1d518969eab529dd9b88c1a")
+	MainnetGenesisHash = common.HexToHash("0x")
+	TestnetGenesisHash = common.HexToHash("0x")
```

Comment:
Added todo. Basically, I am removing the ethereum's trace here. because our genesis hash is not the one in the code. Added todo to populate our own genesis hash. (these hashes are not used anywhere either way for now)

Reply:
Then can we remove them?

Reply:
Done.